### DISCUSSION TITLE
Prettier config's trailing comma option

### DISCUSSION THREAD

**Discussion by hyunbinseo** [sveltejs/kit#10520] [Q&A]
The default template's [Prettier configuration](https://github.com/sveltejs/kit-template-default/blob/1b7aba97d628858745e5dbe29b311717f7a669d7/.prettierrc) sets the trailing comma to none.

```json
{
  "useTabs": true,
  "singleQuote": true,
  "trailingComma": "none",
  "printWidth": 100,
  "plugins": ["prettier-plugin-svelte"],
  "pluginSearchDirs": ["."],
  "overrides": [{ "files": "*.svelte", "options": { "parser": "svelte" } }]
}
```

Is there a specific reason for this?

The default option [changed to](https://prettier.io/docs/en/options#trailing-commas) `"trailingComma": "all"` in Prettier v3.

**Comment by Shyam-Chen** [comment]
This is for my personal use:

```jsonc
{
  "singleQuote": true,
  "printWidth": 100,
  "plugins": ["prettier-plugin-svelte"],
  "pluginSearchDirs": ["."],
  "overrides": [{ "files": "*.svelte", "options": { "parser": "svelte" } }]
}
```
